tests/integration/integration_suite_test.go

```diff
@@ -347,15 +352,40 @@ func testStargate(t *testing.T, namespace string) {
 func TestUpgradeScenario(t *testing.T) {
 	namespace := initializeCluster(t)
 	// Install first production version
-	InstallK8ssandraFromRepo(t, namespace, "1.0.0")
+	DeployClusterWithValues(t, namespace, "default", "cluster_with_reaper.yaml", 1, false, false)
```

Comment:
But this does not actually deploy an old version? Or where is the version defined?

Reply:
I think we might want two tests that is, one from older version, one from the previous stable. Or even one -> stable -> current.

Reply:
Agreed, let me make it so that we can specify a version.